SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -208,7 +208,17 @@ bool FSpatialGDKEditorModule::ShouldStartLocalServer() const
 		return true;
 	}
 
-	return ShouldConnectToLocalDeployment() || (ShouldConnectToLocalDeployment() && ShouldConnectServerToCloud());
+	if (ShouldConnectToLocalDeployment())
+	{
+		// Start the PIE server(s) if we're connecting to a local deployment.
+		return true;
+	}
+	if (ShouldConnectToLocalDeployment() && ShouldConnectServerToCloud())
```

Comment:
I don't really understand this use case? The user is configured to connect to a local deployment, so runtime is running on the local machine, so what is the server connected to the cloud going to do?

Reply:
my bad it is supposed to be cloud deployment :) I will fix it